### PR TITLE
Syndicate surgery bag now has very strong syndicate surgery tools to go with it!

### PR DESCRIPTION
**Pull request by PKPenguin321**
A remake of https://github.com/tgstation/tgstation/pull/31982 which was a remake of https://github.com/HippieStation/HippieStation/pull/4131

Now with more good boy points, travis passing (hopefully), and actual functionality (arm forgot to put the new tools in the bag)

I have no personal thoughts on the balance of the actual objects at this time but feel free to get mad

🆑
add: The Syndicate surgery bag now has very strong syndicate surgery tools to go with it! They do surgeries faster and deal more damage.
/:cl:

### CODE REVIEW

**Comment by Supermichael777** [comment]
This is placed in several tour.. reeducation rooms

**Comment by Armhulen** [comment]
@PKPenguin321 Thanks a billion!

**Comment by Armhulen** [comment]
oh i forgot, make sure to replace hippie's sublime lore with fixium lore. it's canon.

**Comment by PKPenguin321** [comment]
It already says fixium

**Comment by Bawhoppen** [comment]
Buff the cautery too

File: code/modules/surgery/tools.dm [hunk 1]
```diff
@@ -239,3 +239,62 @@
...
+	icon_state = "sscalpel"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 15
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/circular_saw/syndicate
+	name = "syndicate circular saw"
+	desc = "It has a little sub-sentient AI on it to correct errors in the cutting process and guide the blade for extremely efficient cuts."
```
Comment (DoctoreMad) [inline code/modules/surgery/tools.dm]: This is getting a bit too extreme, too quickly.

**Comment by XDTM** [comment]
These are faster than the abductor variants, which imo should be the absolute top due to alien technology. Could you buff those to match?

**Comment by DoctoreMad** [comment]
>buff

Nerf

**Comment by WJohn** [comment]
These sprites don't look terrific, in my opinion. Try upping the contrast of the greys/blacks. You can use my grey ramp if you want.

**Comment by Anonmare** [comment]
Might wanna bring the abductor surgical tools in line with their engineering variants if you're having them this fast. Abductor engineering tools are at toolspeed 0.1 FYI

File: code/modules/surgery/tools.dm (hunk 1 above)
Comment (JJRcop) [reply code/modules/surgery/tools.dm]: ```desc = "Modified with a guidance computer for superhuman precision."```

Comment (Supermichael777) [reply code/modules/surgery/tools.dm]: jjr is corect

**Comment by JJRcop** [comment]
PKPenguin321#2

**Comment by optimumtact** [comment]
this pr is the literal definition of power creep, you should feel bad

👎 from me

File: code/modules/surgery/tools.dm [hunk 2]
```diff
@@ -239,3 +239,62 @@
...
+	icon_state = "sdrill"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 18
+	origin_tech = "materials=3;biotech=3;syndicate=3"
+	toolspeed = 0.3
+
+/obj/item/scalpel/syndicate
+	name = "syndicate scalpel"
+	desc = "It incises effortlessly with it's fixium blade."
```
Comment (QualityVan) [inline code/modules/surgery/tools.dm]: its

**Comment by davethwave** [comment]
perhaps make it spawn without the syndi MMI and then buff the tools keeping the cost considering most people have no use for the syndi MMI and then the buff would be fair for the cost without it but put the buffed tools AND the syndi MMI and make that a role only item for the roboticist perhaps making it cost an extra one or 2 tc for the roboticist considering the usefulness of the syndi MMI in his case.

**Comment by QualityVan** [comment]
this gives you two force 18 weapons for 3tc
buying a dagger gives you one for 2tc
still of the opinion that buying surgery tools should get you surgery tools, not murder tools, and also that uplink shit shouldn't be the best form of something you can find on the station

**Comment by davethwave** [comment]
Also I agree with @QualityVan here, they are used for surgery and not combat, if you want combat get something else.
Perhaps lower their damage and in the surgery bag descryption add (Since these tools were made with precision and surgery in mind the tools can only cut shallowly, even more so with the surgery hardware/software preventing you from cutting too deep as it is a generally bad idea in surgery to do so.)

**Comment by davethwave** [comment]
perhaps instead of damage make then cause the person you use them with harm in mind to bleed quicker instead of doing a lot of damage, like if the person would lose 1% of blood every thirty seconds due to a normal fight, with the SAME amount of damage the person would instead bleed out at 5% of their blood every minute after it and would need to head to medical quickly for patching up and healing/extra blood.

**Comment by Bawhoppen** [comment]
Just make them 15 force and it'll be all good

**Comment by calzilla1** [comment]
I think the whole apeal of the dagger isnt because its a good weapon, but a good STEALTH weapon.

**Comment by Anonmare** [comment]
The dagger is a weapon that almost always evades being tested by security and has a whopping 35 throwforce damage and 100% embed chance

**Comment by PKPenguin321** [comment]
force reduced on scalpel to 12, saw and drill to 15

**Comment by PKPenguin321** [comment]
i can reduce the powercreep by reverting the abductor tools speedup if that's wanted
other than that i think this is pretty much good

File: code/modules/surgery/tools.dm [hunk 3]
```diff
@@ -239,3 +239,62 @@
...
+	toolspeed = 0.3
+
+/obj/item/circular_saw/syndicate
+	name = "syndicate circular saw"
+	desc = "Modified with a guidance computer for superhuman precision."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "ssaw"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	force = 15
```
Comment (Shadowlight213) [inline code/modules/surgery/tools.dm]: This js the same force as the parent. Dont need this line.

**Comment by Supermichael777** [comment]
NO ANONMARE MY POCKET PROTECTOR STRAT

**Comment by calzilla1** [comment]
Can we get a final verdict?

**Comment by PKPenguin321** [comment]
*cough*

**Pull request closed without merging**